tests/testTriggers.go

```diff
@@ -3466,14 +3468,62 @@ func TriggerVolumeDelete(contexts *[]*scheduler.Context, recordChan *chan *Event
 				UpdateOutcome(event, err)
 			})
 
-			ValidateVolumesDeleted(ctx.App.Key, vols)
+			err := ValidateVolumesDeleted(ctx.App.Key, vols)
+			UpdateOutcome(event, err)
+			log.Infof("waiting for 5 mins allowing vols to delete")
+			time.Sleep(5 * time.Minute)
```

Comment:
instead of static 5mins wait, can we have a while loop for 5min which continuously checks for the existence of the LUN, if not exit quicker.

Reply:
fixed